main/parse.c

```diff
@@ -2567,9 +2568,9 @@ static boolean makeKindDescriptionPseudoTag (kindOption *kind,
 	vStringPut (letter_and_name, ',');
 	vStringCatS (letter_and_name, kind -> name);
 
+	d = kind -> description? kind -> description: kind -> name;
```

Comment:
personal style opinion:  I'd find this more readable as
`d = kind->description ? kind->description : kind->name;`